float32 xmin
float32 ymin
float32 xmax
float32 ymax
float64 probability
int16 id
float32 Class